client/src/lib/components/Comment.svelte

```diff
@@ -5,12 +5,17 @@
     import Fa from "svelte-fa";
     import { currentUser } from "../../routes/stores";
     import { fly } from "svelte/transition";
+    import type { ChapterCommentResponse } from "bindings/ChapterCommentResponse";
 
-    export let comment: ComicCommentResponse;
+    export let comment: ComicCommentResponse | ChapterCommentResponse;
     export let indent = 0;
-    let openChildren = false;
+    let openChildren = comment.parent_comment == null;
```

Comment:
problem with this is that if you add a reply under a reply, it's collapsed by default

Reply:
I made it like this to not clutter the screen with nested replies, so only top level replies (not to be confused with comments) will be open.

also when sending a reply I will make it open the reply replied to

Reply:
yeah good idea